packages/pwa/app/partials/product-view/index.jsx

```diff
@@ -137,7 +141,7 @@ const ProductView = ({
                 <Button
                     key="cart-button"
                     onClick={handleCartItem}
-                    disabled={!canOrder}
+                    disabled={showInventoryMessage}
```

Comment:
The error message has a boolean toggle `optionsMessage`, but the add to cart button disable state is based on `showInventoryMessage`, could it be possible that we run into a state where `optionsMessage` does not equal to `showInventoryMessage`?

Reply:
Is it true that when we show an error message, the button should always be disabled?